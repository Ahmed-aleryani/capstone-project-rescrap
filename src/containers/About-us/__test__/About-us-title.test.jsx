import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import AboutUsTitle from '../About-us-title';

it("matches the snapshot", () =>{
    const tree = renderer.create(<AboutUsTitle/>).toJSON();
    expect(tree).toMatchSnapshot();
})