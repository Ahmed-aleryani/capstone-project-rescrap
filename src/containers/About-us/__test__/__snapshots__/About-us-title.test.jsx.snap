// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div
  className="bg-green-200 aboutUsTitle bg-no-repeat bg-auto"
  data-testid="AboutUsTitle"
>
  <h1
    className="text-5xl text-center text-white py-20"
  >
    OUR BUSSINESS
  </h1>
</div>
`;
